pkg/kubectl/cmd/rollout/rollout_history.go

```diff
@@ -88,9 +88,7 @@ func RunHistory(f *cmdutil.Factory, cmd *cobra.Command, out io.Writer, args []st
 		Flatten().
 		Do()
 	err = r.Err()
-	if err != nil {
-		return err
-	}
+	cmdutil.CheckErr(err)
```

Comment:
Also it seems that `rollout history` doesn't have this issue since it's just a function called and its error is passed directly in CheckErr.